litestar/controller.py

```diff
@@ -127,7 +127,7 @@ class Controller:
 
     All route handlers under the controller will have the fragment appended to them. If not set it defaults to ``/``.
     """
-    response_class: type[Response] | None
+    response_class: ResponseType | None
```

Comment:
I think we should standardize on this in the other direction, that is to prefer `type[Response]`.

Reading `type[Response]` is as clear as you can get, but reading `ResponseType` not so much as one doesn't know if it accepts a union of types, or is optional or anything until inspecting the actual alias definition.

For other composite types where we accept complex unions of things there is a benefit of not having to repeat annotations like `HTTPRouteHandler | WebsocketRouteHandler | ASGIRouteHandler` again and again, but here we are talking about 2 extra characters.

Absolutely not a critique of your work, as I know you are just making this consistent with what we are doing elsewhere, but a good time to discuss.

Reply:
Makes sense, I'm fine with dropping the `ResponseType` alias altogether (fyi it touches 6 more files than the PR as of now).

Reply:
> fyi it touches 6 more files than the PR as of now

I'm OK with that.

I had a quick look through the other type aliases that we have, if there were others that were equally redundant then I'd suggest we do it as a separate effort, but this appears to be the only one.

(there are others that I'd question whether they are complex enough, but they are at least more complex than `ThingType = type[Thing]`)

Thanks!